src/plugins/default/DuplicateCheckerPlugin.ts

```diff
@@ -50,16 +53,19 @@ export class DuplicateCheckerPlugin implements IEventPlugin {
     if (merged) {
       merged.incrementCount(count);
       merged.updateDate(context.event.date);
+      context.log.info("Ignoring duplicate event with hash: " + hashCode);
       context.cancelled = true;
       return;
     }
 
     if (this._processedHashcodes.some(h => h.hash === hashCode && h.timestamp >= (now - this._interval))) {
+      context.log.info("Adding event with hash: " + hashCode);
       this._mergedEvents.push(new MergedEvent(hashCode, context, count));
       context.cancelled = true;
       return;
     }
 
+    context.log.info("Enqueueing event with hash: " + hashCode + "to cache.");
```

Comment:
Guess we could add trace (https://github.com/DeveloperToolsWG/console-object/blob/master/api.md#consolelogobject--object-) doesn't look like it's supported everywhere. We'd have to wrap it like we are doing the other ones I guess.